README.md

```diff
@@ -138,5 +45,21 @@ php artisan db:seed
 
 ```
 
-Now you can start collaborating with Dashi. :)
+## Usage
+
+In order to start using Dashi in your daily routine, you need to: 
+
+1. Generate a Slack Incoming WebHook and copy the URL.
```

Comment:
I think that the step 1 should be a link like:
Generate a [Slack Incoming WebHooks](https://slack.com/apps/A0F7XDUAZ-incoming-webhooks)